files/grest/rpc/pool/pool_metadata.sql

```diff
@@ -1,12 +1,26 @@
-CREATE OR REPLACE FUNCTION grest.pool_metadata(_pool_bech32 text)
-RETURNS JSON STABLE LANGUAGE PLPGSQL AS $$
+CREATE OR REPLACE FUNCTION grest.pool_metadata (_pool_bech32 text)
```

Comment:
Since we're touching this query, we could potentially also start adding a null condition that brings back complete table woth pool_bech32 <> metadata url mapping, assuming timings are..not bad

Reply:
😆 Will test and update today

Reply:
Added 👍